src/Seq.Extensions.Logging/Seq/Extensions/Logging/EnrichingEvent.cs

```diff
@@ -0,0 +1,39 @@
+using Serilog.Core;
+using Serilog.Events;
+
+namespace Seq.Extensions.Logging;
+
+/// <summary>
+/// The input to an enricher.
+/// </summary>
+public sealed class EnrichingEvent
```

Comment:
Making this a `struct` seems like it'd save an allocation. Because of the way it's used, I think it'd be fine if the uninitialized default threw `NullReferenceException` arbitrarily when its methods were called.